Build/MQTTnet.AspNetCore.nuspec

```diff
@@ -28,6 +28,14 @@
         <dependency id="MQTTnet" version="$nugetVersion" />
       </group>
     </dependencies>
+    <frameworkReferences>
+      <group targetFramework=".NETCoreApp3.1">
```

Comment:
@JanEggers Could you please verify that this is correct?

Reply:
I dont know whats the benefit of depending on a frameworkreference. it worked without them. my preference would be to remove the nuspec altogether and just use the csproj to build the packages. is there any info in the nuspec that cannot be provided by csproj?

Reply:
For now I want to keep the nuspec due to the build script and release notes etc. etc. But for the future we can surely think about the generated packages from VS. So you say we can remove these references?